sm/sm.c

```diff
@@ -476,6 +476,12 @@ void m0_sm_state_set(struct m0_sm *mach, int state)
 	M0_PRE(m0_sm_invariant(mach));
 	state_set(mach, state, 0);
 }
+
+void m0_sm_state_and_rc_set(struct m0_sm *mach, int state, int32_t rc)
+{
+	M0_PRE(m0_sm_invariant(mach));
+	state_set(mach, state, rc);
+}
 M0_EXPORTED(m0_sm_state_set);
```

Comment:
You want to add a similar M0_EXPORTED line for the function m0_sm_state_and_rc_set

Reply:
Ok. I will update it.